presto-hive/src/main/java/com/facebook/presto/hive/HivePageSourceProvider.java

```diff
@@ -547,10 +554,10 @@ public int getIndex()
         }
 
         /**
-         * @param columns columns that need to be returned to engine
-         * @param requiredInterimColumns columns that are needed for processing, but shouldn't be returned to engine (may overlaps with columns)
+         * @param columns                   columns that need to be returned to engine
```

Comment:
nit: why this formatting?

Reply:
it was automatically reformatted.  I think because it makes the descriptions all line up.